base/params/list.go

```diff
@@ -10,11 +28,48 @@ var ListAll = List{Limit: -1}
 // List defines input parameters for listing operations
 type List struct {
 	Filter  []string
-	OrderBy []string
+	OrderBy OrderBy
```

Comment:
Can we add json struct tags to serialize all of the fields in this struct so that they must be lower-case? Makes it more consistent with how everything else works.

Reply:
yessss thank youuuuu